src/test/java/org/gridsuite/filter/server/ExpertFilterUtilsTest.java

```diff
@@ -1,560 +0,0 @@
-        andRules2.add(andCombination);
-        NumberExpertRule numRule4 = NumberExpertRule.builder().value(20.0)
-                .field(FieldType.TARGET_V).operator(OperatorType.EQUALS).build();
-        andRules2.add(numRule4);
-        EnumExpertRule enumRule4 = EnumExpertRule.builder().value("HYDRO")
-                .field(FieldType.ENERGY_SOURCE).operator(OperatorType.EQUALS).build();
-        andRules2.add(enumRule4);
-        EnumExpertRule enumRule5 = EnumExpertRule.builder().value("OTHER")
-                .field(FieldType.ENERGY_SOURCE).operator(OperatorType.NOT_EQUALS).build();
-        andRules2.add(enumRule5);
-        StringExpertRule stringRule5 = StringExpertRule.builder().value("ID")
-                .field(FieldType.ID).operator(OperatorType.BEGINS_WITH).build();
-        andRules2.add(stringRule5);
-        BooleanExpertRule booleanRule6 = BooleanExpertRule.builder().value(true)
-                .field(FieldType.VOLTAGE_REGULATOR_ON).operator(OperatorType.EQUALS).build();
-        andRules2.add(booleanRule6);
-        BooleanExpertRule booleanRule7 = BooleanExpertRule.builder().value(false)
-                .field(FieldType.VOLTAGE_REGULATOR_ON).operator(OperatorType.NOT_EQUALS).build();
-        andRules2.add(booleanRule7);
-
-        CombinatorExpertRule andFilter = CombinatorExpertRule.builder().combinator(CombinatorType.AND).rules(andRules2).build();
-
-        boolean result = andFilter.evaluateRule(gen);
-
-        assertTrue(result);
-    }
-
-    @Test
-    void testEvaluateExpertFilterWithFalseANDCombination() {
-        List<AbstractExpertRule> andRules1 = new ArrayList<>();
-        NumberExpertRule numRule1 = NumberExpertRule.builder().value(0.0)
-                .field(FieldType.MIN_P).operator(OperatorType.LOWER).build();
-        andRules1.add(numRule1);
-        NumberExpertRule numRule2 = NumberExpertRule.builder().value(105.0)
-                .field(FieldType.MAX_P).operator(OperatorType.GREATER_OR_EQUALS).build(); // false
-        andRules1.add(numRule2);
-        CombinatorExpertRule andCombination = CombinatorExpertRule.builder().combinator(CombinatorType.AND).rules(andRules1).build();
-
-        List<AbstractExpertRule> andRules2 = new ArrayList<>();
-        andRules2.add(andCombination);
-        NumberExpertRule numRule3 = NumberExpertRule.builder().value(20.0)
-                .field(FieldType.TARGET_V).operator(OperatorType.EQUALS).build();
-        andRules2.add(numRule3);
-        EnumExpertRule enumRule4 = EnumExpertRule.builder().value("HYDRO")
-                .field(FieldType.ENERGY_SOURCE).operator(OperatorType.EQUALS).build();
-        andRules2.add(enumRule4);
-
-        CombinatorExpertRule andFilter = CombinatorExpertRule.builder().combinator(CombinatorType.AND).rules(andRules2).build();
-
-        boolean result = andFilter.evaluateRule(gen);
-
-        assertFalse(result);
-    }
-
-    @Test
-    void testEvaluateExpertFilterWithORCombination() {
```

Comment:
Same

Reply:
Idem, rule tree is tested by https://github.com/gridsuite/filter-server/pull/90/files#diff-b4e0eb7fb6e066e9822531fb175401d400663c91679b6a7c2a936fe2fd7e74ccR48